### DISCUSSION TITLE
Which KNX topology is required to use Tasmota KNX

### DISCUSSION THREAD

**Discussion by avrandrei** [arendst/Tasmota#13371] [Q&A] [answered]
Hi! I have a pretty simple KNX installation in my home: only one line using the TP KNX medium. In my installation I also have a KNX IP Router which I used for downloading my ETS configuration and for integrating my KNX system with Home Assistant.

I would like to extend my system using some Tasmota KNX devices: mainly sonoffs which will respond and send their states in the system and I'm wondering if there's anything I need to change in ETS. How do I know if the IP router is multicasting and can be used with Tasmota KNX? Do I need to configure it as a backbone coupler?

Right now if I look in ETS all the structure is using TP (backbone, area, line). Should I change this?

**Comment by ascillato** [comment, marked as answer]
You don't need ETS for using Tasmota. Besides, Tasmota don't support being configured from ETS yet. You can use a dummy device in ETS and then do the config manually in the Tasmota device.

What you need is go directly to Config Web Page of the KNX IP Router and see if the Multicast option is enabled (by default it is enabled in most brands) and that's it. You just then need to go to Tasmota UI and in the KNX menu set to which Group Address Tasmota should send commands and which Group Address Tasmota should listen.

https://tasmota.github.io/docs/KNX/
